File: Source/Bifrost.JavaScript.Specs/utils/for_StringMapping/when_matching_a_string_with_wildcard_at_the_end.js
describe("when matching a string with wildcard at the end", function () {
    var mapping = Bifrost.StringMapping.create({
        format: "{something}/**/",
        mappedFormat: "whatevva.**."
    });
    var result = mapping.matches("this/is/a/wildcard/uri");

    it("should match", function () {
        expect(result).toBe(true);
    });
});

File: Source/Bifrost.Specs/Utils/for_StringMapping/when_matching_a_string_with_wildcard_at_the_end.cs
using Machine.Specifications;

namespace Bifrost.Utils.for_StringMapping   
{
    public class when_matching_a_string_with_wildcard_at_the_end
    {
        static StringMapping mapping = new StringMapping(
                "{something}/**/",
                "{something}.**."
            );

        static bool result;

        Because of = () => result = mapping.Matches("this/is/a/wildcard/string");

        It should_match = () => result.ShouldBeTrue();
    }
}
